scripts/boot_patch.sh

```diff
@@ -173,7 +173,7 @@ if [ -n "$PREINITDEVICE" ]; then
   echo "PREINITDEVICE=$PREINITDEVICE" >> config
 fi
 [ -n "$SHA1" ] && echo "SHA1=$SHA1" >> config
-RANDOMSEED=$(tr -dc 'a-f0-9' < /dev/urandom | head -c 16)
+RANDOMSEED=$(tr -dc 'a-f0-8' < /dev/urandom | head -c 16)
```

Comment:
?

Reply:
It's explained in the notes... It was suggested that was intended as a hex number by the 0x prefix in the next line, so I limited it correctly to the hexadecimal charset... I can remove that change if that's not what you actually wanted... 👍

Reply:
so... hex numbers do not contain 9?

Reply:
Ah, right you are. My brain broke this evening. 😆